### PR TITLE
Add more details around carts vs images

### PR DESCRIPTION
**Pull request by danmcp**

### CODE REVIEW

**Comment by danmcp** [comment]
@adellape Review please.

File: whats_new/carts_vs_images.adoc [hunk 1]
```diff
@@ -15,20 +15,22 @@ Cartridges in OpenShift v2 were the focal point for building applications.  Each
...
 
 == Collocation
 Whereas cartridges in OpenShift v2 could be collocated within gears, images are mapped 1:1 with containers. Containers use pods as their collocation mechanism.
 
 == Source Code
+In OpenShift v2, applications were required to have at least 1 web framework with 1 git repo.  With OpenShift v3, you can choose which images are built from source and that source can be located outside of OpenShift itself.  Because the source is disconnected from the images, the choice of image and source are distinct operations with source being optional.
 
 == Build
+In OpenShift v2 builds happened in place as part of you application gears.  This meant downtime for non scaled applications due to resource constraints.  In v3, builds happen in separate containers.
```
Comment (adellape) [inline whats_new/carts_vs_images.adoc]: Typo:

s/you application/your application/

Suggestion:

s/non scaled/non-scaled/

**Comment by adellape** [comment]
@danmcp Just the one comment inline, LGTM otherwise.

**Comment by danmcp** [comment]
@adellape Fixed

**Pull request merged**